reverse-proxy-nginx/package-conf-secure/http-minio-secure.conf

```diff
@@ -0,0 +1,31 @@
+server {
```

Comment:
Just a question, aren't we going to consider the insecure mode ?

Also, I want to highlight that the tests of monitoring package should be updated

Reply:
Yes, thanks I missed adding the insecure config. Which tests do you mean?

Reply:
You can check test/cucumber folder in the root of the project, now the CI CD pipelines will execute the tests there.
So these tests are for single and cluster mode and it will run the the packages that got a change on it, it will check if the services of that packages are up and running and also when destroy will check if all services, volumes and configs were removed

It is not a big change that should be done in both: test/cucumber/features/cluster-mode/kafka-packages.cluster.feature
and also test/cucumber/features/single-mode/kafka-packages.feature

to run it:
- single : cd  test/cucumber && yarn && yarn test:single:kafka 
- cluster: cd test/cucumber && yarn test:cluster:kafka